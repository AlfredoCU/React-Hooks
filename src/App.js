import React from "react";
// import Counter from "./components/01-useState/Counter";
// import CounterWithCustomHooks from "./components/01-useState/CounterWithCustomHooks";
// import Form from "./components/02-useEffect/Form";
// import FormWithCustomHooks from "./components/02-useEffect/FormWithCustomHooks";
// import Quotes from "./components/03-examples/Quotes";
// import FocusScreen from "./components/04-useRef/FocusScreen";
// import RealExampleRef from "./components/04-useRef/RealExampleRef";
// import { Layout } from "./components/05-useLayoutEffect/Layout";
// import Memo from "./components/06-useMemo/Memo";
// import MemoHook from "./components/06-useMemo/MemoHook";
// import CallBack from "./components/07-useCallback/Callback";
// import { Padre } from "./components/08-tarea-memo/Padre";
// import TodoApp from "./components/09-useReducer/TodoApp";
import MainApp from "./components/10-useContext/MainApp";
// import "./components/09-useReducer/intro-reducer";

const App = () => {
  return (
    <div className="app">
      <header className="app-header">
        <h1>Hooks</h1>
      </header>
      <main>
        {/* <Counter />
        <CounterWithCustomHooks />
        <Form />
        <FormWithCustomHooks />
        <Quotes />
        <FocusScreen />
        <RealExampleRef />
        <Layout />
        <Memo />
        <MemoHook />
        <CallBack />
        <Padre /> */}
        {/* <TodoApp /> */}
        <MainApp />
      </main>
    </div>
  );
};

export default App;
