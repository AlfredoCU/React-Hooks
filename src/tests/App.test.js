import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import App from "../App";

describe("Pruebas al componente App", () => {
  test("Foto del componente", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
