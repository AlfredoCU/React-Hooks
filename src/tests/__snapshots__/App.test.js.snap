// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente App Foto del componente 1`] = `
<div
  className="app"
>
  <header
    className="app-header"
  >
    <h1>
      Hooks
    </h1>
  </header>
  <main>
    <MainApp />
  </main>
</div>
`;
